website/content/en/docs/building-operators/helm/migration.md

```diff
@@ -23,9 +23,12 @@ The `build/Dockerfile` directory was replaced by the `Dockerfile` in the root di
 
 ### What is new
 
-- Now users are able to use [kustomize][kustomize] in the configurations files
-- PROJECT file in the root directory has all information about the project
-- Users are able to customize commands for your own projects via the Makefile which is added on the root directory
+To bring a little context, projects are now scaffold using
+
+- [kustomize][kustomize] to perform the configurations 
+- Makefile with helpers which brings more flexibility and allows customizations
+- Protect by default via [kube-auth-proxy][kube-auth-proxy] 
+- Prometheus metrics configured via [kustomize][kustomize] 
```

Comment:
Being a little pedantic here. Prometheus metrics themselves aren't configured via kustomize. However the metrics server, kube-rbac-proxy, and service monitor are:
```suggestion
- Updated metrics configuration using [kube-auth-proxy][kube-auth-proxy], a `--metrics-addr` flag, and [kustomize][kustomize]-based deployment of a Kubernetes `Service` and prometheus operator `ServiceMonitor`
```